release/release_tests.yaml

```diff
@@ -1707,21 +1707,31 @@
     wait_for_nodes:
       num_nodes: 3
 
-
-  smoke_test:
-    frequency: nightly
-
-    run:
-      timeout: 3600
-
   alert: long_running_tests
 
+  variations:
+    - __suffix__: aws
+    - __suffix__: smoke-test
+      frequency: nightly
+      run:
+        timeout: 3600
+        script: python workloads/apex.py
+        long_running: true
+        wait_for_nodes:
+          num_nodes: 3
```

Comment:
I made a mistake on my sample PR too, the correct way to do this today is to remove this block, keep the existing smoke_test block, and add another smoke_test block into [1]